styles.css

```diff
@@ -17,7 +17,7 @@
             "left-content"
             "middle-content"
             "aside-right"
-            "empty-space"
+            "none"
```

Comment:
Je hebt nu een component "none" gemaakt en in de grid geplaatst. Dit is niet nodig. Je kan inplaats hiervan een "." In je grid-template-area gebruiken op de plek waar je geen component wilt plaatsen.

Reply:
Ah ok bedankt voor de tip!